index.html

```diff
@@ -6,7 +6,31 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Metadata Management Tool</title>
     <link rel="stylesheet" type="text/css" href="/static/src/css/initial/index.scss">
+    <!-- Google Tag Manager -->
+    <script>
+      (function(w, d, s, l, i) {
+        w[l] = w[l] || [];
+        w[l].push({
+          'gtm.start':
+                  new Date().getTime(),
+          event: 'gtm.js'
+        });
+        var f = d.getElementsByTagName(s)[0],
+                j = d.createElement(s),
+                dl = l != 'dataLayer' ? '&l=' + l : '';
+        j.async = true;
+        j.src = 'https://www.googletagmanager.com/gtm.js?id=' + i + dl;
+        f.parentNode.insertBefore(j, f);
+      })(window, document, 'script', 'dataLayer', 'GTM-WNP7MLF');
```

Comment:
I'm not sure if you want to be committing an ID like this into the repo.

And do you want to report to Google in the dev environment?

Reply:
I don't think there is any harm in committing the ID but it is nice to pull those values from a config to make local development/debugging easier.

Though, I assume we probably want this set up similarly to Earthdata Search. When in development we initialize the development GA instance directly, without tag manager. And when in SIT, UAT, or Prod initialize the appropriate Tag Manager container (GTM-WNP7MLF), which will load the appropriate GA instance for the particular environment.

Reply:
@htranho - Just so you're aware, I made some edits to the comment above after looking deeper at the code.

Reply:
Updated: GA4 ID as passed in variable, please review.